packages/ui-icons/package.json

```diff
@@ -30,6 +30,8 @@
     "src:icons:storybook": "babel-node --config-file ./babel-builder.config.js builder/builder-story.js --output-dir ./src --icons-dir ./src"
   },
   "dependencies": {
+    "classnames": "^2.2.6",
+    "@path-composer/ui-icons": "file:../ui-icons"
```

Comment:
is this a correct name for the package?

Reply:
yes, this is an advanced monorepo implementation supported by Lerna